google/ggrc-core
# GGRC-6786/GGRC-6810 Create external comment model

# Issue description

GGRCQ need to create ggrc id for comment. To avoid conflicts we need to introduce new object "external_comment". These comments will be created by external app only.
 external_comment should be searchable via query API.

# Steps to test the changes

* Check if api requests work proper for Control Comments (POST/PUT/GET/DELETE)
* Check export of Control
* Check api that is based on revisions
* Check fulltext functionality

# Solution description

Create new model ExternalComment and mixin ExternalCommentable. Replace Commentable mixin with ExternalCommentable.

# Sanity checklist


# Migration checklist


# PR Review checklist
